internal/api/v1/service/index.go

```diff
@@ -56,10 +57,13 @@ func makeResponse(ctx context.Context, appsOf map[string][]string, services serv
 	response := models.ServiceResponseList{}
 
 	for _, service := range services {
-
 		serviceDetails, err := service.Details(ctx)
 		if err != nil {
-			return models.ServiceResponseList{}, err
+			if apierrors.IsNotFound(err) {
```

Comment:
You can keep it together and avoid the `continue`

```golang
if err != nil && !apierrors.IsNotFound(err){
    return models.ServiceResponseList{}, err
}
```

Reply:
good one, fixing

Reply:
Hm, I need the continue because I don't want the rest of the code in this block to run

Reply:
Ah, I haven't see the rest of the block. Then it's fine :+1: